frontend/src/routes/MonthlyWorkingHours.jsx

```diff
@@ -2,26 +2,36 @@ import { useEffect, useState } from "react";
 import axios from "axios";
 import { Table, Col, Row } from "antd";
 import SideBar from "../components/SideBar";
+import { ResponsiveBar } from "@nivo/bar";
 
 const MonthlyWorkingHours = () => {
   const [data, setData] = useState([]);
   const [selectedYear, setSelectedYear] = useState(2024);
+  const [selectedYearsBack, setSelectedYearsBack] = useState(3);
+  const [dataGraph, setDataGraph] = useState([]);
 
   const onYearChange = (e) => {
     setSelectedYear(e.target.value);
   };
 
+  const onYearsBackChange = (value) => {
```

Comment:
This function is unnecessary, because it takes the same argument as the function it calls and simply calls it. Could use setSelectedYearsBack directly in the SideBar props.